src/processor/snapshot/exporter.rs

```diff
@@ -116,6 +121,8 @@ impl SnapshotExporter {
         let mut has_more = true;
 
         while has_more {
+            tracing::info!("Serializing chunk {chunk_id}...");
```

Comment:
would be more informative if it also contained the total count...

Reply:
I agree, do you know if it's possible to get the total count of rows from a RocksDB column? I skimmed the docs quickly but couldn't find anything

Reply:
apparently not - at least not exactly (see e.g. https://stackoverflow.com/questions/68340346/querying-the-size-of-a-column-family-in-rocksdb )...

Reply:
Added a commit that displays the total count, fetches it from the last repeated key index instead.